0000-notes-on-oer-encoding/0000-notes-on-oer-encoding.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Notes on OER Encoding
+draft: 1
+---
+
+# Introduction
```

Comment:
I think it would be useful to structure this article like a Wirecutter review. The top should say very quickly what we picked and why it's great, then it should explain why you should trust this decision and go into the details.